data/complete.queryMap.json

```diff
@@ -108,6 +108,7 @@
+  "4c30eb7c3732d005b5537e60c0c419dd": "query CommercialButtonsTestsQuery {\n  artwork(id: \"artworkID\") {\n    ...CommercialButtons_artwork\n    id\n  }\n}\n\nfragment CommercialButtons_artwork on Artwork {\n  slug\n  internalID\n  is_acquireable\n  is_offerable\n  is_inquireable\n}\n",
```

Comment:
😭 someday I want to resurrect https://github.com/artsy/emission/pull/1300 to fix these, if anyone else is interested!